pkg/apps/watch-appstream.py

```diff
@@ -141,7 +141,7 @@ def try_size(sz):
         path = os.path.join(dir, "..", "icons", origin, sz, icon)
         return path if os.path.exists(path) else None
 
-    return try_size("64x64") or try_size("128x128")
+    return try_size("scalable") or try_size("64x64") or try_size("128x128")
```

Comment:
My feeling is that png support is much more straight forward in most browsers so we should prefer png to svg. But maybe that is wrong.

@mvollmer, do you have an opinion?

Reply:
Let's not change the cached icon search logic at all, as there is no way in practice to get the collection data generators to create the `/usr/share/app-info/$origin/icons/scalable` directory.  I think.

Reply:
Let's not change the cached icon search logic at all, as there is no way in practice to get the collection data generators to create the `/usr/share/app-info/$origin/icons/scalable` directory. I think.